cdap-common/src/main/java/co/cask/cdap/common/ServiceBindException.java

```diff
@@ -24,8 +25,8 @@
   private final String host;
   private final int port;
 
-  public ServiceBindException(String service, String host, int port) {
-    super(String.format("%s failed to bind to %s:%d", service, host, port));
+  public ServiceBindException(String service, String host, int port, Exception cause) {
```

Comment:
Adding this because otherwise the root exception is lost (see callers of this constructors).